content/english/java-basics/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Java: Basics"
+description: "An introduction to the basics of Java programming language."
+date: 2020-07-07T00:00:00Z
+prereq: "none"
+difficulty: "Beginner"
+icon: "fab fa-java"
+draft: false
+---
+
+## Overview
+
+<img src="images/logo.png" height="150"/> 
+
+Welcome to the course <b>Java: Basics</b>, where you will learn a widely used high-level programming language that gives computer instructions to create exciting programs!
```

Comment:
**Java: Basics** for bold instead of the HTML tags.

Reply:
Similarly elsewhere in the files.

Reply:
I tried using ** ** for bold text, but it doesn't show up as bold when I run the website on Hugo server while <b> tags works.

Reply:
That's weird... have you tried going to the other pages that use ** and see **bold** text?